### PR TITLE
Create demo.html

### PR DESCRIPTION
**Pull request by 1130945**
Added development file

### CODE REVIEW

File: demo.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<P>My Dev File</P>
```
Comment (nbharadwaj138) [inline demo.html:1]: Starting Reeview

Comment (nbharadwaj138) [reply demo.html:1]: Review Completed

**Review by nbharadwaj138 — APPROVED** [review]
Completed

**Comment by 1130945** [comment]
Merged the changes

**Pull request merged**